star-wars-browser/src/store/people/reducer.js

```diff
@@ -1,10 +1,10 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { initialState } from './defaultState';
-import { setPeopleData } from './actions';
+import { setPeople } from './actions';
 
 const peopleReducer = createReducer(initialState, (builder) => {
   builder
-    .addCase(setPeopleData, (state, action) => {
+    .addCase(setPeople, (state, action) => {
       // eslint-disable-next-line no-param-reassign
```

Comment:
możesz to przerzucić do pierwszej linii (komentarz zmieniasz na /* */ i usuwasz `-next-line`

Reply:
poprawione